DigikalaCrawler/spiders/comment_spider.py

```diff
@@ -64,8 +66,16 @@ def parse_comments(self, response):
             pos = [x.strip() for x in pos_raw]
             neg_raw = comment_selector.css('.c-comments__evaluation-negative li::text').extract()
             neg = [x.strip() for x in neg_raw]
-            txt = comment_selector.css('p::text').extract_first()
+            
+            ##########MODIFIED BY ARMAN############
+            # txt = comment_selector.css('p::text').extract_first()
+            txt_list = comment_selector.css('p::text').extract()
+            txt = ""
+            for txt_itr in txt_list:
+                txt += txt_itr.strip()+"\n"
```

Comment:
I think it makes sense to concatenate those lines with single whitespace instead of `"\n"`.